scripts/info.sh

```
@@ -20,7 +35,11 @@ get_circuit_size() {
     popd > /dev/null
 }
 
+# Check the versions of Noir and BB
+check_versions
+
 cd $SCRIPT_DIR/../examples
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)